ui/tests/helpers/secret-engine/secret-engine-helpers.js

```diff
@@ -9,6 +9,16 @@ import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engin
 import { stringArrayToCamelCase } from 'vault/helpers/string-array-to-camel';
 import { v4 as uuidv4 } from 'uuid';
 
+/* Secret Create/Edit methods */
+// ARG TODO unsure if should be moved to another file
```

Comment:
Could rename the file to `secret-helpers` instead and then it's more broad in scope but I think it makes sense to be here since creating an engine and secret are present in many workflows.

Reply:
I created this file when I was working on adding configuration endpoints and pages to aws, gcp and azure. It became very configuration focused, but the naming indicates it should have a broader scope. I suspect I may end up breaking this up in to two files and we continue with the plugin work, but solid point about naming.

For now @drivera258 I think this is good and I'll keep that TODO as a note to revisit as we progress.